chidioguejiofor/airtech-api
# #164838899: Sets up django testing

#### What does this PR achieve?
- installs pytest-django and pytest
- adds pytest.ini config to the app
- adds a catch-all endpoint and tests adds tests for it
- installs pytest-cov
- adds CircleCi config
- updates the README with CI Badges
- adds a codeclimate.yml file

#### Any background context?
- Setups pipenv shell 
- installs `yapf` package

#### New environmental Variables?
- `DATABASE_URI`: a URI that points to the database
- `SECRET_KEY`: the app secret

#### Manual Testing Steps
- Fetch branch via `git fetch ch-setup-testing-164838899`
- Checkout branch via `git checkout ch-setup-testing-164838899`
- Run tests via `pytest`
- All tests should run and pass successfully

#### Screenshots:
<img width="1440" alt="Screenshot 2019-03-27 at 11 17 01 AM" src="https://user-images.githubusercontent.com/38565349/55068337-e38a3680-5081-11e9-9b15-675e2d98b07e.png">

#### Any new deployment instructions
- None
